tiledb/sm/query/query.cc

```diff
@@ -804,7 +807,8 @@ Status Query::cancel() {
 }
 
 Status Query::process() {
-  if (status_ == QueryStatus::UNINITIALIZED)
+  if (status_ == QueryStatus::UNINITIALIZED ||
+      status_ == QueryStatus::INITIALIZED)
     return logger_->status(
         Status_QueryError("Cannot process query; Query is not initialized"));
```

Comment:
This error message is super confusing given the new condition is called INITIALIZED.